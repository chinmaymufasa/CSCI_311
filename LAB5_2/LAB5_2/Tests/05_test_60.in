2
73
5
2
106
6
2
145
4
2
102
5
2
51
6
2
91
4
2
161
6
2
186
5
2
156
4
2
145
6
2
133
6
2
75
6
2
124
6
2
132
6
2
16
6
9
3
106
6
9
2
86
4
2
120
5
2
189
6
2
172
4
2
114
4
9
3
815
4
9
-1
