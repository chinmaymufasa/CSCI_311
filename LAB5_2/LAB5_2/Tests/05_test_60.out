73 
106 73 
106 73 145 
73 102 106 145 
51 102 73 145 106 
102 73 51 91 106 145 
51 91 73 106 161 145 102 
51 73 91 102 106 145 161 186 
102 73 51 91 145 106 161 156 186 
51 91 73 145 106 156 186 161 145 102 
51 91 73 106 145 133 156 186 161 145 102 
51 75 91 73 106 145 133 156 186 161 145 102 
51 75 91 73 124 106 145 133 156 186 161 145 102 
51 75 91 73 106 132 124 145 133 156 186 161 145 102 
16 51 75 91 73 106 132 124 145 133 156 186 161 145 102 
15
16 51 75 91 73 132 124 145 133 156 186 161 145 102 
14
102 73 51 16 86 75 91 145 133 124 132 145 161 156 186 
16 51 73 75 86 91 102 120 124 132 133 145 145 156 161 186 
16 51 75 91 86 73 120 132 124 145 133 156 189 186 161 145 102 
102 73 51 16 86 75 91 145 133 124 120 132 145 161 156 186 172 189 
102 73 51 16 86 75 91 145 124 120 114 133 132 145 161 156 186 172 189 
19
102 73 51 16 86 75 91 145 124 120 114 133 132 145 161 156 186 172 189 
19
